src/main/rx/fport.c

```diff
@@ -151,7 +153,7 @@ static void fportDataReceive(uint16_t c)
 
     const timeUs_t currentTimeUs = micros();
 
-    if (bufferPosition > 0 && cmpTimeUs(currentTimeUs, frameStartAt) > FPORT_TIME_NEEDED_PER_FRAME_US + 500) {
+    if (!skipUntilStart && (frameStarting || bufferPosition > 0) && cmpTimeUs(currentTimeUs, frameStartAt) > FPORT_TIME_NEEDED_PER_FRAME_US + 500) {
```

Comment:
`frameStarting` is set after receiving `FPORT_FRAME_MARKER` at end of packet - this will timeout in interpacket space

It also causes problem that reply data is received as frame (because it's sent after marker)

Reply:
Well spotted, will fix.